test/integration/addSprite.js

```diff
@@ -27,8 +27,8 @@ test('default cat', t => {
     vm.on('playgroundData', e => {
         const threads = JSON.parse(e.threads);
         t.ok(threads.length === 0);
+        vm.quit();
```

Comment:
Can this be run in a tap test clean up function instead? E.g. `t.afterEach()` or `t.teardown()`?

Reply:
In this case it could be, since `const vm` is at the outermost scope, but in several of the other tests the `vm` is created as a local variable so an `afterEach` or `teardown` won't have access to it.

In this particular file, only this one test starts the VM so I figured it should only `quit` in the one test. That way other tests will timeout & fail if they accidentally cause the VM to start or otherwise open a handle unexpectedly.